model/aws/registry.yaml

```diff
@@ -481,3 +488,96 @@ groups:
           and [upload-part-copy](https://docs.aws.amazon.com/cli/latest/reference/s3api/upload-part-copy.html) operations.
           The `part_number` attribute corresponds to the `--part-number` parameter of the
           [upload-part operation within the S3 API](https://docs.aws.amazon.com/cli/latest/reference/s3api/upload-part.html).
+  - id: registry.aws.sqs
```

Comment:
if we need any aws.sqs attributes on top of generic messaging, please add a span definition for it and reference all applicable attributes, see [kafka yaml](https://github.com/open-telemetry/semantic-conventions/blob/5b24784e032ada99312d5a17d6970da434f5ce88/model/messaging/spans.yaml#L138) and [md](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/messaging/kafka.md) conventions as an example.

Reply:
Probably won't need any more attributes so will remove this section.